include/tsl/sparse_hash.h

```diff
@@ -1343,25 +1399,28 @@ class sparse_hash : private Allocator,
    */
   iterator begin() noexcept {
     auto begin = m_sparse_buckets_data.begin();
-    while (begin != m_sparse_buckets_data.end() && begin->empty()) {
+    //fancy pointers seem to have a problem with "->" in this context
```

Comment:
point out where it is because of std::vector

Reply:
Comments have been updated.